<?php

try {
    if (rand(0, 1)) {
        throw new Exception("OcurriÃ³ un error!");
    }
    echo "Todo bien!";
} catch (Exception $e) {
    echo "ExcepciÃ³n capturada: " . $e->getMessage();
}
